code/modules/reagents/reagents/toxins.dm

```diff
@@ -515,6 +515,7 @@
 	M.stats.addTempStat(STAT_VIG, -STAT_LEVEL_BASIC * effect_multiplier, STIM_TIME, "pararein")
 	M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_BASIC * effect_multiplier, STIM_TIME, "pararein")
 	M.stats.addTempStat(STAT_COG, STAT_LEVEL_ADEPT * effect_multiplier, STIM_TIME, "pararein")
+	M.add_chemical_effect(CE_TOXIN, strength)
```

Comment:
wew getting hit with 6 strength twice

Reply:
does it do both this act_effect AND the parent?

Reply:
yes

Reply:
lol oops, i'll- go ahead and take away this one